test.ml

```diff
@@ -52,6 +52,6 @@ let loop filename tc () =
 
 (* part 2 *)
 let () =
-  if String.equal Sys.argv.(1) "-t"
```

Comment:
New in Ocaml 4.03. For some reason 4.03 binaries not released for Ubuntu yet http://packages.ubuntu.com/search?keywords=ocaml